internal/initialize/security_test.go

```diff
@@ -59,9 +60,8 @@ func TestPodSecurityContext(t *testing.T) {
 		assert.Assert(t, psc.RunAsUser == nil,
 			`Containers must not set runAsUser to 0`)
 
-		// TODO(cbandy): delegate to v1.SecurityContext
 		assert.Assert(t, psc.SeccompProfile == nil,
```

Comment:
🤔 I think I was suggesting we do an assertion similar to `psc.RunAsNonRoot` a few lines up.

Reply:
Like this?
```go
if assert.Check(t, psc.SeccompProfile == nil) {
  assert.Assert(t, initialize.RestrictedSecurityContext().SeccompProfile != nil,
    `SeccompProfile should be delegated to the container-level v1.SecurityContext`)
}
```

Reply:
Yes! Thanks!